src/webui/www/private/scripts/mocha-init.js

```diff
@@ -144,6 +144,19 @@ var initializeWindows = function() {
             height: loadWindowHeight(id, 420),
             onResize: function() {
                 saveWindowSize(id);
+            },
+            onClose: function() {
+                var hashString = location.hash && location.hash.replace(/^#/, '');
```

Comment:
And IE11 does permit `const` & `let` so you are encouraged to use them whenever possible.